### ISSUE TITLE
[RNN] Create RNN model with tensorflow

### ISSUE THREAD

**Issue by wrbbz** [ArsenyZorin/RNNCodeClones#5]
Create RNN model with python and tensorflow. Train it with seq2seq outputs

**Comment by wrbbz** [comment]
Created Siamese RNN with contrastive loss function
